### PR TITLE
Completed adding the ability for users to see bills by their selected representative.

### PR DESCRIPTION
**Pull request by ryancurrie**
apella-show-rep-bills

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -68,6 +68,32 @@ MongoClient.connect('mongodb://localhost/apella', (err, db) => {
         })
       })
   })
+
+  app.get('/bills-by-rep/:repId', ({ params: { repId } }, res) => {
```
Comment (ronperris) [inline index.js]: How about `/rep/:id/bills/`?

File: index.js [hunk 2]
```diff
@@ -68,6 +68,32 @@ MongoClient.connect('mongodb://localhost/apella', (err, db) => {
...
+      .then((resp, err) => {
+        if (err) {
+          return res.sendStatus(500)
+        } else {
+          res.status(200).send(resp.body.results[0].bills)
+        }
+      })
+  })
+
+  app.get('/get-rep-by-id/:repId', ({ params: repId }, res) => {
```
Comment (ronperris) [inline index.js]: `/rep/:id`

File: index.js [hunk 3]
```diff
@@ -68,6 +68,32 @@ MongoClient.connect('mongodb://localhost/apella', (err, db) => {
...
+    representatives.findOne({ id: repId.repId }, (err, resp) => {
```
Comment (ronperris) [inline index.js]: repId is an object with a repId property?

**Review by ronperris — CHANGES_REQUESTED** [review]
Check the comments.

File: index.js (hunk 3 above)
Comment (ryancurrie) [reply index.js]: Yes, I needed only the property value.

Comment (ronperris) [reply index.js]: Ah, you're renaming params to repId. I think what you want it `{ params: { repId }}`.  Then you can do `findOne({ id: repId})`.

File: index.js (hunk 1 above)
Comment (ryancurrie) [reply index.js]: This is done

File: index.js (hunk 3 above)
Comment (ryancurrie) [reply index.js]: This is done

File: index.js (hunk 1 above)
Comment (ryancurrie) [reply index.js]: So update the route name?

**Pull request merged**